tox.ini

```diff
@@ -20,18 +20,21 @@ envlist =
 
 [testenv]
 commands =
-    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest --lsof {posargs}
+    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest {env:_PYTEST_TOX_ARGS} {posargs}
     coverage: coverage combine
     coverage: coverage report
 passenv = USER USERNAME COVERAGE_* TRAVIS
 setenv =
+    _PYTEST_TOX_ARGS=--lsof
```

Comment:
i suspect this needs a comment that taking off lsoff for xdist is intended

we still need some thing to run lsoff against tho

Reply:
`--lsof` was not used with the previous `py27-xdist` / `py37-xdist` jobs, that's why I've removed it here.  Will try to keep it in general.